src/devices/LidarLiteV3/PowerOptionFlag.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.TimeOfFlight.Models.LidarLiteV3
+{
+    /// <summary>
+    /// Power option flags
+    /// </summary>
+    [Flags]
+    public enum PowerOptionFlag
```

Comment:
I think most of the devices call this PowerMode or similar (at least remove `Flag` suffix if you prefer to keep this name)

Reply:
will fix